src/controllers/controller.bar.js

```diff
@@ -220,6 +220,23 @@ export default class BarController extends DatasetController {
 		return parsed;
 	}
 
+	/**
+	 * @protected
+	 */
+	getMinMax(scale, canStack) {
+		const result = super.getMinMax(scale, canStack);
+
+		for (let i = 0; i < this._cachedMeta.data.length; i++) {
+			const custom = this.getParsed(i)._custom;
+			if (custom) {
+				result.min = Math.min(custom.min, result.min);
+				result.max = Math.max(custom.max, result.max);
+			}
```

Comment:
Could we add an `else` here to handle the non-floating case and avoid the call to `super.getMinMax`? I think that would perform better than always having to loop over the data twice

Reply:
Would have to duplicate the stacking and skipping logic, to support zoom etc.

Now that I said that, this actually breaks the zoom support.